#commands
## servo_index is from 0
uint8 SET_SERVO_HOMING_OFFSET = 1
# [servo_index, joint_offset] !torque disable!
uint8 SET_SERVO_PID_GAIN = 2
# [servo_index, p_gain, i_gain, d_gain]
uint8 SET_SERVO_PROFILE_VEL = 3
# [servo_index, profile_vel]
uint8 SET_SERVO_SEND_DATA_FLAG = 4
# [servo_index, flag]
uint8 SET_SERVO_CURRENT_LIMIT = 5
# [servo_index, current_limit] !torque disable!
uint8 SET_DYNAMIXEL_TTL_RS485_MIXED = 6
# [flag]
uint8 SET_SERVO_EXTERNAL_ENCODER_FLAG = 7
# [servo_index, flag]
uint8 SET_SERVO_RESOLUTION_RATIO = 8
# [servo_index, joint_resolution, servo_resolution]


uint8 command
int32[] data
---
bool success
float32[] data 
